### ISSUE TITLE
Layers button textures don't update properly

### ISSUE THREAD

**Issue by DaylightDr3amer** [Orama-Interactive/Pixelorama#404]
**Pixelorama version:**
v0.8.2

**OS/device including version:**
Linux Debian 10

**Issue description:**
The buttons in the layer don't change icons after a theme change.
This only happens in the layer's buttons and not in the other's.

**Steps to reproduce:**
- Change from dark theme to a different type of theme (light theme)
- Click the hide/lock/link button
- The textures should be taken from the dark theme instead of the correct one

**Comment by DaylightDr3amer** [comment]
SOLVED: add these lines of code in res://src/UI/Timeline/LayerButton.gd line 19:
```gdscript
# Reload button texture themes after they're created (or reloaded, see res://src/Classes/Project.gd:layers_changed())
	for button in [visibility_button, lock_button, linked_button]:
		var texture: TextureRect = button.get_child(0)
		var last_backslash = texture.texture.resource_path.get_base_dir().find_last("/")
		var button_category = texture.texture.resource_path.get_base_dir().right(last_backslash + 1)
		var normal_file_name = texture.texture.resource_path.get_file()
		var theme_type := Global.theme_type
		if theme_type == Global.Theme_Types.CARAMEL or (theme_type == Global.Theme_Types.BLUE and button_category != "tools"):
			theme_type = Global.Theme_Types.DARK

		var theme_type_string : String = Global.Theme_Types.keys()[theme_type].to_lower()
		texture.texture = load("res://assets/graphics/%s_themes/%s/%s" % [theme_type_string, button_category, normal_file_name])```

**Comment by OverloadedOrama** [comment]
The issue has been fixed in ab583a7354ff33a9961377748bc52d73970ec770 :)
